src/v/features/feature_table.h

```diff
@@ -350,6 +351,12 @@ constexpr static std::array feature_schema{
     "cloud_metadata_cluster_recovery",
     feature::cloud_metadata_cluster_recovery,
     feature_spec::available_policy::always,
+    feature_spec::prepare_policy::always},
+  feature_spec{
+    cluster::cluster_version{11},
+    "audit_logging",
+    feature::audit_logging,
+    feature_spec::available_policy::always,
```

Comment:
Like GSSAPI (and I think some other features), I think we want to make sure we 'nag' the user if this is enabled and they don't have a license

Reply:
I'm pretty sure that is only for features which are "enterprise only" that are enabled without a license.

Reply:
Isn't auditing enterprise only?

Reply:
Confirmed this is the case, +1 will change